glib/src/error.rs

```diff
@@ -82,6 +83,12 @@ impl Error {
                 .unwrap_or_else(|err| str::from_utf8(&bytes[..err.valid_up_to()]).unwrap())
         }
     }
+
+    pub fn into_raw(mut self) -> *mut ffi::GError {
```

Comment:
I think documentation is needed here, considering that it's calling "mem::forget" on `self`. Also: shouldn't this be an `unsafe` function?

Reply:
No, doesn't have to be `unsafe`